pocs/linux/kernelctf/CVE-2023-4147_lts_cos/docs/exploit.md

```diff
@@ -0,0 +1,377 @@
+### Triggering Vulnerability
```

Comment:
In mitigation version of exploit (https://github.com/google/security-research/pull/111/files?short_path=d6ecf7c) one could find steps to reproduce vulnerability. Could you add the instructions to reproduce the vulnerability in this PR too please?

Reply:
Yes, I added the reproduce step to the exploit.md.